update laravel version, add codecov

kibbonz opened the pull request.

karl-d reviewed (changes requested):
Changes looking all good. Just need some changes to actually output the coverage file to be able to upload to codecode. Review the output of the actions.

I have updated the global token that it needed to be able to upload, but now it is missing the output.

kibbonz commented:
> Changes looking all good. Just need some changes to actually output the coverage file to be able to upload to codecode. Review the output of the actions.
> 
> I have updated the global token that it needed to be able to upload, but now it is missing the output.

fixed
